src/crypto/auth/hmacsha512.rs

```diff
@@ -48,19 +172,137 @@ mod test {
         // corresponding to tests/auth.c from NaCl
         // "Test Case 2" from RFC 4231
         assert!(::init());
-        let key = [0x4a, 0x65, 0x66, 0x65, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-                   0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-                   0x00, 0x00, 0x00, 0x00, 0x00, 0x00];
-        let c = [0x77, 0x68, 0x61, 0x74, 0x20, 0x64, 0x6f, 0x20, 0x79, 0x61, 0x20, 0x77, 0x61,
-                 0x6e, 0x74, 0x20, 0x66, 0x6f, 0x72, 0x20, 0x6e, 0x6f, 0x74, 0x68, 0x69, 0x6e,
-                 0x67, 0x3f];
+        let key = [
```

Comment:
I think it's better to skip rustfmt changes for these cases - e.g. the way [used here](https://github.com/maidsafe/crust/blob/6c86a5a5e3574a331a3c0c049e0549220d10ce9f/src/main/bootstrap/mod.rs#L186).
The array formatting is certainly horrible in this case with the annoying vertical stretch.